### PR TITLE
feat: add preset configuration to babel options

### LINKED ISSUE #54: presets for babel
Since this plugin requires us to stop using babelrc (and I assume babel.config.js), how can we list additionnal presets to be used (such as @babel/typescript, @babel/preset-react ?

Thanks !

### PR DESCRIPTION
**Pull request by semoal**
resolves #54

### CODE REVIEW

**Comment by semoal** [comment]
@DanielSchaffer ci is crashing but i'm not sure why, thanks in advance mate

**Comment by DanielSchaffer** [comment]
Looks like I might need to disable code coverage on external branches? I might just disable it completely for now I haven’t written any unit tests 🤦‍♂️🙊

**Comment by DanielSchaffer** [comment]
Looks like this is actually a somewhat common issue with coveralls. Go ahead and remove the the `- yarn coverage-report` line from `.travis.yml`. I'll figure a better solution out later.

File: src/babel-target.ts [hunk 1]
```diff
@@ -245,14 +243,14 @@ export class BabelTargetFactory {
     return {
       presets: [
         ['@babel/preset-env', mergedPresetOptions],
+        ...this.presets,
```
Comment (DanielSchaffer) [inline src/babel-target.ts:246]: Have you checked this when `preset` is not set? Node generally doesn't like destructuring `undefined`

Comment (DanielSchaffer) [reply src/babel-target.ts:246]: Actually, looks like you set it to an empty array in the main plugin - in that case, it shouldn't be an optional param.

**Review by DanielSchaffer — CHANGES_REQUESTED** [review]

**Comment by semoal** [comment]
@DanielSchaffer Fixed your proposal changes mate, once release i'll migrate my current webpack conf to yours, so probably i'll push more "fixes"

**Review by DanielSchaffer — APPROVED** [review]

**Comment by DanielSchaffer** [comment]
Available in prerelease [v2.5.0-next.2](https://github.com/DanielSchaffer/webpack-babel-multi-target-plugin/releases/tag/v2.5.0-next.2)

**Comment by barroudjo** [comment]
awesome, thanks !

**Pull request merged**